### PR TITLE
coreboot-configurator: 2022-08-22 -> 2023-01-17, use pkexec instead of sudo.

### PR DESCRIPTION
**Pull request by r-vdp**
###### Description of changes

Bump version.

Use pkexec instead of sudo, as in the upstream code. Using pkexec makes it possible to launch the application using its .desktop file, instead of needing to run it from the terminal.

For usage on non NixOS systems, we add an argument for the pkexec path. On such systems, the polkit policy files will also need to be included in the system's config.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/coreboot-configurator/default.nix [hunk 1]
```diff
@@ -3,34 +3,43 @@
...
   };
 
   nativeBuildInputs = [ inkscape meson ninja pkg-config wrapQtAppsHook ];
   buildInputs = [ yaml-cpp qtbase qtsvg ];
 
   postPatch = ''
     substituteInPlace src/application/*.cpp \
-      --replace '/usr/bin/pkexec' 'sudo' \
-      --replace '/usr/bin/systemctl' 'systemctl' \
+      --replace '/usr/bin/pkexec' '${pkexecPath}' \
```
Comment (danth) [inline pkgs/tools/misc/coreboot-configurator/default.nix:34]: Would it be possible to load this from `$PATH`? That would avoid needing to pass in the absolute path as an argument.

Comment (R-VdP) [reply pkgs/tools/misc/coreboot-configurator/default.nix:38]: @danth good point. I tested this and it seems to work indeed, so I put the path by default now but I left the possibility to override it if needed.

**Review by danth — APPROVED** [review]
Looks good to me!

**Comment by Lassulus** [comment]
Result of `nixpkgs-review pr 217842` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>coreboot-configurator</li>
  </ul>
</details>

**Pull request merged**